setup.py

```diff
@@ -191,18 +234,20 @@ def test_otio():
     },
     extras_require={
         'dev': [
-            'flake8==3.5',
-            'coverage==4.5',
-            'tox==3.0',
+            'flake8>=3.5',
+            'coverage>=4.5',
+            'tox>=3.0',
         ],
         'view': [
-            'PySide2==5.11'
+            'PySide2>=5.11'
```

Comment:
My inclination for floating versions is to use the [compatible release](https://www.python.org/dev/peps/pep-0440/#compatible-release) version specifier (`~=`). This way we can opt into minor and patch release without opting in to majors that could have breaking changes.

Reply:
Theoretically, in semantic versioning, if a major version is upped, it would be a breaking change. So if dependency developers are observing that rule a major version bump means work needs to be done for that dependency to keep working.
Whether or not that's observed is another question. And I would agree, for the things in dev my inclination is to get those to break as soon as possible.

So maybe I want to alter my recommendation. For `view` `~=` might make sense, for `dev` `>=` pushes us to stay up to date, and that's a good thing.

Reply:
Ok fair enough.  I'll switch view to be ~=.  I think that would be more in line with the VFX platform.